packages/host/src/AppDrawer.tsx

```diff
@@ -130,10 +134,13 @@ const Menu: React.FC<MenuProps> = ({ classes }) => {
 const AppDrawer: React.FC = () => {
   const classes = useStyles();
   const drawer = useDrawer();
+  const t = useTranslation();
 
   return (
     <Drawer
+      data-testid="drawer"
```

Comment:
argument for `data-testid` is that you should generally try not to, but if you can't access the element then if you use a class name, or css id then you're using implementation details, so just use `data-testid` 🤷